src/Symfony/Component/Validator/Constraints/Unique.php

```diff
@@ -33,12 +35,19 @@ class Unique extends Constraint
     public $normalizer;
 
     public function __construct(
-        array $options = null,
+        array|string $fields = null,
```

Comment:
I'm not sure it's a good idea to add the argument first. It would make BC safer if this were moved last

Reply:
> I'm not sure it's a good idea to add the argument first. It would make BC safer if this were moved last

Ok, moved to the end. One can use named arguments for that.

The failing checks are not related to this PR.